.github/workflows/master.yml

```diff
@@ -178,9 +176,10 @@ jobs:
         with:
           images: scadalts/scadalts
       - name: Build and push Docker image
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v5
         with:
           context: .
+          platforms: linux/amd64,linux/arm64
```

Comment:
Isn't it enough to set this parameter? I understand that the simplest solution is for a single runner and this will lengthen the process of adding individual images, but we want to limit the number of changes to at least necessary to avoid potential problems later.

Reply:
Yes, that's correct, if you want to limit the changes made you only need to setup QEMU, buildx and then build-and-push with that parameter